static/js/drafts.js

```diff
@@ -74,12 +75,23 @@ export const draft_model = (function () {
 
     exports.editDraft = function (id, draft) {
         const drafts = get();
+        let changed = false;
+
+        function check_if_equal(draft_a, draft_b) {
+            // eslint-disable-next-line no-unused-vars
+            const {updatedAt: unused_a, ...a} = draft_a;
+            // eslint-disable-next-line no-unused-vars
+            const {updatedAt: unused_b, ...b} = draft_b;
```

Comment:
Can you explain this algorithm?  Is the intent that we just want to compare these values after dropping the `updatedAt` field?  https://stackoverflow.com/questions/37922967/exclude-some-properties-in-comparison-using-isequal-of-lodash seems like a much better way to do that.

Reply:
Merged after replacing this.